addons/backpacks/CfgVehicles.hpp

```diff
@@ -0,0 +1,29 @@
+class CfgVehicles {
+    class Man;
+    class CAManBase: Man {
+        class ACE_SelfActions {
+            class ACE_Equipment {
+                class ACE_BackpackToFront {
+                    displayName = "$STR_ACE_Backpack_BackpackToFront";
+                    condition = "(backpack _player != '') and (vehicle _player == _player) and (isNull (_player getVariable ['ACE_Backpack', objNull]))";
```

Comment:
vehicle _player == _player

can be removed. checked as trivial condition anyway

exception would be "isNotInside"